frontend/components/timer/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from 'react';
+
+export const Timer = ({ time = 180 }) => {
+    const [seconds, setSeconds] = useState(time);
+
+    useEffect(() => {
+        let interval = setInterval(() => {
+            if(seconds >= 1){
+                setSeconds(seconds => seconds - 1)
+            }
+            else if(seconds === 0){
+                alert("Time is over!")
+                clearInterval(interval)
+            }
+        }, 1000)
```

Comment:
Small note here, we don't really need the check for `seconds >= 1`. When the `seconds` hit zero, we clear the interval. Therefore we could do something like:
```javascript
if (seconds === 0) {
    alert("Time is over!");
    clearInterval(interval);
}

setSeconds(seconds - 1);
```

Reply:
True, thanks for the feedback :-)